apiserver/facades/client/application/deployrepository.go

```diff
@@ -292,6 +295,8 @@ func makeDeployFromRepositoryValidator(ctx context.Context, cfg validatorConfig)
 	v := &deployFromRepositoryValidator{
 		charmhubHTTPClient: cfg.charmhubHTTPClient,
 		model:              cfg.model,
+		modelInfo:          cfg.modelInfo,
+		modelConfigService: cfg.modelConfigService,
```

Comment:
Seems strange to pass model info and then a service. Should we just pass the config or does that not make sense here?

Reply:
Previously I was passing in a model info service, you asked me to replace this with static model info.

The model config service is later passed in to the charmRepoFactory. It needs to access the charmhub URL. I don't think we should replace this with static config, then it will not be able to pick up updates to the charmhub URL.